community/cypher/cypher-compiler-3.0/src/main/scala/org/neo4j/cypher/internal/compiler/v3_0/planner/logical/steps/planShortestPaths.scala

```diff
@@ -83,39 +83,43 @@ case object planShortestPaths {
     val lhs = lpp.planApply(inner, lhsOption)
 
     val pattern = shortestPath.rel
+    val rhsArgument = lpp.planArgumentRowFrom(lhs)
 
-    // TODO: Decide the best from and to based on degree (generate two alternative plans and let planner decide)
-    // (or do bidirectional var length expand)
-    val from = pattern.left
+    val rhs = if (context.errorIfShortestPathFallbackUsedAtRuntime) {
+      lpp.planError( rhsArgument, new ShortestPathFallbackDisabledException )
+     } else {
```

Comment:
I'm not finding the warning that was requested if we had the flag set to false. That should also be coded here, right?